t/012-multi.t

```diff
@@ -1,10 +1,17 @@
-use Test; # -*- mode: perl6 -*-
+use Test;
```

Comment:
I really need the mode lines, but, hey, not a big deal.

Reply:
Maybe we should switch to `.rakutest` extensions then?

Reply:
Probably.